tests/test_commands.py

```diff
@@ -1115,7 +1115,7 @@ def test_linsert(self, r):
         assert r.lrange('a', 0, -1) == [b'1', b'2', b'2.5', b'3']
         assert r.linsert('a', 'before', '2', '1.5') == 5
         assert r.lrange('a', 0, -1) == \
-            [b'1', b'1.5', b'2', b'2.5', b'3']
+               [b'1', b'1.5', b'2', b'2.5', b'3']
```

Comment:
Is the green highlight a tab? If so - and for the others can we replace tabs with spaces. If not.. I'll see why it's highlighting this way.

Reply:
It looks like spaces